src/popup/router/pages/Index.vue

```diff
@@ -38,68 +44,124 @@
 
 <script>
 import { IN_FRAME } from '../../utils/helper';
-import Logo from '../../../icons/logo.svg?vue-component';
+import AnimatedSpinner from '../../../icons/animated-spinner.svg?skip-optimize';
 import SuperheroLogo from '../../../icons/superhero-logo.svg?vue-component';
 import CheckBox from '../components/CheckBox';
 import Button from '../components/Button';
 import Platforms from '../components/Platforms';
 
 export default {
-  components: { Logo, SuperheroLogo, CheckBox, Button, Platforms },
+  components: { SuperheroLogo, CheckBox, Button, Platforms },
   data: () => ({
     termsAgreed: false,
     IS_WEB: process.env.PLATFORM === 'web',
     IN_FRAME,
+    AnimatedSpinner,
   }),
 };
 </script>
 
 <style lang="scss" scoped>
-@import '../../../styles/variables';
+@import '../../../styles/typography';
 
 .index {
-  .checkbox-container {
-    margin: 0 auto 25px auto;
-    max-width: 282px;
-    font-size: 15px;
+  margin-top: -10px;
+  text-align: center;
+
+  .terms-agreement {
+    margin: 0 auto 24px auto;
+    max-width: 292px;
+    display: flex;
+
+    @extend %face-sans-15-medium;
+
+    .checkbox-container {
+      margin-right: 4px;
+      color: $color-dark-grey;
+
+      &.agreed {
+        color: white;
+      }
 
-    ::v-deep .checkmark {
-      margin-right: 5px;
+      &:hover:not(.agreed),
+      &:active:not(.agreed) {
+        transition: all 0.12s ease-in-out;
```

Comment:
Maybe we can use single 
```
    & > * {
      transition: all 0.12s ease-in-out;
    }
```
in parent `terms-agreement` and get rid of multiply transition? For me it will look equal